extensions/wikia/AdEngine/js/AdConfigMobile.js

```diff
@@ -32,14 +32,14 @@ define('ext.wikia.adEngine.adConfigMobile', [
 			return [taboola];
 		}
 
-		if (instantGlobals && instantGlobals.wgSitewideDisableGpt) {
-			return [];
-		}
-
 		if (context.providers.turtle) {
 			return [turtle, remnantGptMobile];
 		}
 
+		if (instantGlobals && instantGlobals.wgSitewideDisableGpt) {
```

Comment:
Is there any other reason of moving this `if` block than alphabetical order? :smile_cat: